src/test/java/com/epam/third/task/comparators/SphereComparatorTest.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.third.task.comparators;
+
+import com.epam.third.task.entities.SphereWithId;
+import com.epam.third.task.logic.IdGenerator;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Comparator;
+
+public abstract class SphereComparatorTest {
+    private final static IdGenerator GENERATOR = new IdGenerator();
+    private final static int SMALLER_ID = GENERATOR.generateNextId();
+    private final static int BIGGER_ID = GENERATOR.generateNextId();
+    private final static SphereWithId EVERYTHING_SMALLER = new SphereWithId(SMALLER_ID, 3, 2, 2, 2);
```

Comment:
create a universal object that suits any test it's a good idea, there is always a test case where your object won't work, so this approach is too rigid and fragile, create a separate test for each comparator with own data